src/views/GovernanceReviewTeam/Summary/index.tsx

```diff
@@ -167,53 +166,19 @@ const RequestSummary = ({
             <Breadcrumb current>{t('itGovernanceRequestDetails')}</Breadcrumb>
           </BreadcrumbBar>
 
-          {/* Request summary */}
-          <div className="display-flex flex-align-end flex-wrap margin-bottom-2">
-            <h2 className="margin-top-05 margin-bottom-0 margin-right-2">
-              {requestName}
-            </h2>
-            <p className="margin-y-05 text-primary-light">
-              {t('submittedOn', {
-                date: formatDateLocal(submittedAt, 'MM/dd/yyyy')
-              })}
-            </p>
-          </div>
-
-          <Grid row gap>
-            <Grid tablet={{ col: 8 }}>
-              <h5 className="text-normal margin-y-0">{t('requestType')}</h5>
-              <h4 className="margin-top-05 margin-bottom-2">
-                {translateRequestType(requestType)}
-              </h4>
-
-              <h5 className="text-normal margin-y-0">
-                {t('systemServiceContractName')}
-              </h5>
-              <h4 className="margin-top-05 margin-bottom-2">
-                {getSystemOrContractName(relationType, contractName, systems)}
-              </h4>
-            </Grid>
-
-            <Grid tablet={{ col: 4 }}>
-              <h5 className="text-normal margin-y-0">
-                {t('intake:contactDetails.requester')}
-              </h5>
-              <h4 className="margin-top-05 margin-bottom-2">
-                {getPersonNameAndComponentAcronym(
-                  requester?.name || '',
-                  requester?.component
-                )}
-              </h4>
-
-              <h5 className="text-normal margin-y-0">
-                {t('intake:review.contractNumber')}
-              </h5>
-              <h4 className="margin-top-05 margin-bottom-2">
-                {/* TODO: (Sam) review */}
```

Comment:
Just highlighting this to keep the tab. I'm assuming this could have to do with `.contractNumbers.map(c => c.contractNumber)` happening in a few places, and attempting to parse that before the frontend gets it? @aterstriep @samoddball 

I removed the todo comment since it wasn't referring to anything else.

We can leave it or make a note somewhere in the sprint to get back to it.

Reply:
i can't even remember. it could be that or how should we be displaying multiple (i.e., bullets, comma separated as they are now, some other format), but i think it's fine to just delete the comment. thanks Adam!